sentinelone/info.json

```diff
@@ -1,3793 +1,3459 @@
-          "editable": true,
-          "visible": true,
-          "type": "checkbox",
-          "name": "tenant",
-          "value": false,
-          "tooltip": "Select this checkbox to indicate a tenant scope in the query."
-        },
-        {
-          "title": "Query Type",
-          "description": "(Optional) Type of the query used by deep visibility in SentinelOne.",
-          "required": false,
-          "editable": true,
-          "visible": true,
-          "type": "text",
-          "name": "queryType",
-          "placeholder": "e.g. events",
-          "tooltip": "Type of the query used by deep visibility in SentinelOne."
-        },
-        {
-          "title": "Account IDs",
-          "description": "(Optional) List of comma-separated agent's accounts based on which you want to retrieve the query ID from SentinelOne.",
-          "required": false,
-          "editable": true,
-          "visible": true,
-          "type": "text",
-          "name": "accountIds",
-          "tooltip": "List of comma-separated agent's accounts.",
-          "placeholder": "e.g 225494730938493804,225494730938493915"
-        },
-        {
-          "title": "Site IDs",
-          "description": "(Optional) List of comma-separated agent's sites based on which you want to retrieve the query ID from SentinelOne.",
-          "required": false,
-          "editable": true,
-          "visible": true,
-          "type": "text",
-          "name": "siteIds",
-          "tooltip": "List of comma-separated agent's sites.",
-          "placeholder": "e.g 225494730938493804,225494730938493915"
-        }
-      ],
-      "enabled": true,
-      "output_schema": {
-        "queryId": "",
-        "queryModeInfo": {
-          "lastActivatedAt": "",
-          "mode": ""
-        }
-      }
-    },
-    {
-      "operation": "cancel_running_query",
-      "title": "Cancel Running Query",
-      "description": "Stops a deep visibility query that is running on SentinelOne based on the query ID you have specified.",
-      "category": "investigation",
-      "annotation": "cancel_running_query",
-      "parameters": [
-        {
-          "title": "Query ID",
-          "description": "The ID of a deep visibility query that you want to stop in SentinelOne. When you create a query in SentinelOne you get its QueryID.",
-          "required": true,
-          "editable": true,
-          "visible": true,
-          "type": "text",
-          "name": "queryId"
-        }
-      ],
-      "enabled": true,
-      "output_schema": {
-        "success": ""
-      }
+    "name": "sentinelone_dev",
+    "version": "3.1.8",
+    "label": "SentinelOne",
+    "description": "SentinelOne that provides threat detection, hunting, and response features that enable organizations to discover vulnerabilities and protect IT operations. This connector facilitates automated operations related to events, threats, agents, query, and applications.",
+    "publisher": "",
```

Comment:
```suggestion
    "publisher": "Fortinet",
```

Reply:
It seems that you have replaced the info.json file. Because of which many keys are missing. Request you to fork the develop branch and add only two operations.